kf_hungarian_tracker/kf_hungarian_tracker/kf_hungarian_node.py

```diff
@@ -185,9 +187,10 @@ def callback(self, msg):
                 marker.scale = obs.msg.size
                 marker_list.append(marker)
                 # make an arrow
+                arrow_id = 2**32 - self.current_rviz_marker_id # descending from 32 bit id
                 arrow = Marker()
                 arrow.header = msg.header
-                arrow.id = 255 - obs.msg.id
+                arrow.id = arrow_id 
```

Comment:
This is for visualization - my memory on `Marker` is a little wavering, but don't you have the "namespace" you can use to make a marker into `ns/id`? If so, you can make the `id` to `0` always and have the `namespace` by the `uuid` of the message. Then there's no 32 bit shifting or marker maps or any of that stuff. Test to make sure though.

Reply:
Yes, you are right. ```Marker``` has a namespace field. Thanks